### DISCUSSION TITLE
How do python-fastdds work with multiple network interfaces  ?

### DISCUSSION THREAD

**Discussion by hl-shi** [eProsima/Fast-DDS-python#180] [Q&A] [answered]
please give help

**Comment by Mario-DL** [comment, marked as answer]
Hi @hl-shi,

Thanks for using Fast DDS Python. Please, refer to the [Fast DDS documentation section](https://fast-dds.docs.eprosima.com/en/latest/fastdds/transport/interfaces.html) in this regard. Fast DDS Python provide language bindings but the behavior is determined by the underlying Fast DDS.

**Reply by hl-shi** [reply]
Thank you so much.  I have solved the problem
